sql/core/src/test/scala/org/apache/spark/sql/execution/CoalesceShufflePartitionsSuite.scala

```diff
@@ -66,6 +66,7 @@ class CoalesceShufflePartitionsSuite extends SparkFunSuite with BeforeAndAfterAl
         .setAppName("test")
         .set(UI_ENABLED, false)
         .set(IO_ENCRYPTION_ENABLED, enableIOEncryption)
+        .set(IO_COMPRESSION_CODEC.key, "lz4")
```

Comment:
Curious, does this test require it to be lz4 ?
Same for `AdaptiveQueryExecSuite` below - why not rely on the new default value of zstd ?